### ISSUE TITLE
Promotion in ascending order

### ISSUE THREAD

**Issue by Nordlandia** [cutechess/cutechess#538]
Based on occurrences and probability. It's good idea to have the promotion window in ascending order with Queen - Knight - Rook - Bishop.  

So ideally the best order is **♕♘♖ ♗**

**Comment by alwey** [comment]
The order of available pieces may appear randomized but they are actually ordered by their occurence in the move list ;-)  .

The piece chooser does not know about specific piece types. And take into account variants, there are other piece types.

![prom1](https://user-images.githubusercontent.com/6425738/67621966-84052c80-f804-11e9-8296-edaa59258415.png)

**Comment by Nordlandia** [comment]
Maybe it's good idea to adjust it for standard chess.

**Comment by alwey** [comment]
Maybe, but it should be optional. Personally, I like the Q-R-B-N better.

**Comment by Nordlandia** [comment]
When it comes to underpromotions. Knight trumps the rook by a considerable margin. So why you prefer Q-R vs over Q-N is a question to me. 

![Promotion](https://user-images.githubusercontent.com/26534454/67622721-23361e00-f81d-11e9-970d-cb3b18d48119.png)

**Comment by alwey** [comment]
Right, I prefer it because Q-R-B-N is the ranking of values. I knew that N has the second highest conversion rate but I did not know the numbers.

> Edit: perhaps opt for arrangement of choise. Edit the list the way one want. Make it customizable.
 
This is what I had in mind.

**Comment by Nordlandia** [comment]
Sounds great. 

Apropos this alway i'm hoping for 7-piece syzygy adjudication. It will save even more than that currrent 6-piece adj. For engine testers this will be very time saving feature.
